### PR TITLE
feat: add cat painting intermediate CSS project

### PR DESCRIPTION
**Pull request by Ksound22**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Wow, this looks awesome. I think it'll be an awesome project to replace the Picasso Painting.

I filled in the spreadsheet based on what you have here, and found a couple of things:

- `transform: rotate()`: This is actually the second project to use this property. The first is the Rothko Painting on [step 43](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-the-css-box-model-by-building-a-rothko-painting/step-43). However, it isn't really introduced, and an example isn't shown. Since this property was used in a much earlier project, it should be fine to use here. Ideally we'd also add a brief introduction to that step in the Rothko Painting project, too.
- `i` elements and Font Awesome: The current Picasso Painting project uses `i` elements for some Font Awesome icons, which are loaded from an external resource. Since we're trying not to use any external resources, it's fine that this element / Font Awesome isn't covered in this project.

Otherwise, one other thing I noticed is that the cat's head is a bit small. I believe the smallest screen we account for is 375 x 667 pixels or so (iPhone SE). If it's possible to make the drawing a bit larger to around 250 x 250 or 300 x 300, that'd be great.

Finally, there's not a lot of contrast between the dark fur and the other colors like the inner ear, nose, and whiskers. It would be great if there was a bit more contrast between the main fur color and the smaller elements, like if the nose and inner ears were pink, the whiskers white, and so on. Or the main fur color could be lightened. There are a lot of options.

I think as long as the colors pass the "Graphical Objects and User Interface Components" here, we should be good: https://webaim.org/resources/contrastchecker/

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+body {
+  background-color: #dfdfe2;
+}
+
+.cat-container {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto 0;
+}
```
Comment (scissorsneedfoodtoo) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:16]: Forgot to mention this earlier, but there doesn't seem to be an element with this class. These can be removed if they're not going to be used:

```suggestion
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 2]
```diff
@@ -0,0 +1,224 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #dfdfe2;
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:8]:
```suggestion
 background-color: #c9d2fc;
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+}
+
+.cat-head {
+  position: absolute;
+  right: 0;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+  background: linear-gradient(#2a2a40 85%, #3b3b4f 100%);
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:25]:
```suggestion
  +  background: linear-gradient(#5e5e5e 85%, #45454f 100%);
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+  position: absolute;
+  top: -29px;
+  left: -31px;
+  z-index: 1;
+  transform: rotate(-45deg);
+  border-top-left-radius: 90px;
+  border-top-right-radius: 10px;
+  border-left: 35px solid transparent;
+  border-right: 35px solid transparent;
+  border-bottom: 70px solid #2a2a40;
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:42]:
```suggestion
  border-bottom: 70px solid #5e5e5e;
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+  position: absolute;
+  top: -29px;
+  left: 144px;
+  z-index: 1;
+  transform: rotate(45deg);
+  border-top-left-radius: 90px;
+  border-top-right-radius: 10px;
+  border-left: 35px solid transparent;
+  border-right: 35px solid transparent;
+  border-bottom: 70px solid #2a2a40;
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:55]:
```suggestion
 border-bottom: 70px solid #5e5e5e;
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+}
+
+.cat-left-eye {
+  position: absolute;
+  top: 54px;
+  left: 39px;
+  transform: rotate(25deg);
+  width: 30px;
+  height: 40px;
+  background-color: #393737;
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:91]:
```suggestion
  background-color: #000;
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+}
+
+.cat-right-eye {
+  position: absolute;
+  top: 54px;
+  left: 116px;
+  transform: rotate(-25deg);
+  width: 30px;
+  height: 40px;
+  background-color: #393737;
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:102]:
```suggestion
  background-color: #000;
```

File: responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+  position: absolute;
+  top: 108px;
+  left: 75px;
+  border-top-left-radius: 50%;
+  border-bottom-right-radius: 50%;
+  border-bottom-left-radius: 50%;
+  transform: rotate(180deg);
+  border-left: 15px solid transparent;
+  border-right: 15px solid transparent;
+  border-bottom: 20px solid #3b3b4f;
```
Comment (zairahira) [inline responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/styles.css:138]:
```suggestion
  border-bottom: 20px solid #442c2c;
```

**Review by zairahira — COMMENTED** [review]
I have made a few changes to the color contrast!

image

**Comment by Ksound22** [comment]
> I have made a few changes to the color contrast!
> 
> image

Thanks, Zaira. It looks a lot better.

**Comment by Ksound22** [comment]
> 

I have enlarged the cat's head. Could you please check it?

**Comment by jdwilkin4** [comment]
@Ksound22 

I think the changes look good.

If @scissorsneedfoodtoo or @zairahira don't have any other comments then I vote we merge this in

**Review by jdwilkin4 — APPROVED** [review]

**Comment by zairahira** [comment]
The changes look good to me, let's merge this in!

**Pull request merged**